### DISCUSSION TITLE
Feature which I miss a lot -- Android n iOS app

### DISCUSSION THREAD

**Discussion by amgschnappi** [photoprism/photoprism#4334] [Polls]
It would be great to have an app which auto uploads certain folders from mobile to your servers.
Would then be a great google photos replacement.

**Comment by sgflt** [comment]
No need to reinvent the wheel. There are already some imeplemntations, f.e. [PhotoSync](https://play.google.com/store/apps/details?id=com.touchbyte.photosync)

**Reply by lastzero** [reply]
There's also a Gallery for PhotoPrism app available for Android and we have contributors working on an iOS app specifically for PhotoPrism. It's still under development and thus not available in the App Store yet. See our Developer Guide in the native apps section.

**Reply by amgschnappi** [reply]
> No need to reinvent the wheel. There are already some imeplemntations, f.e. [PhotoSync](https://play.google.com/store/apps/details?id=com.touchbyte.photosync)

Some advt infested app. No thx. Willing to pay a one time money for official PP or PP endorsed app.

**Reply by lastzero** [reply]
The android app is free as in free software and in free beer.

**Reply by amgschnappi** [reply]
> There's also a Gallery for PhotoPrism app available for Android and we have contributors working on an iOS app specifically for PhotoPrism. It's still under development and thus not available in the App Store yet. See our Developer Guide in the native apps section.

Nice app. Thx. 
No import / auto upöoad facility. Also wud instill confidence if it comes under officiall PP brand.

**Reply by lastzero** [reply]
It's open source and the maintainer works closely with us. I guess you could contribute upload functionality (or use another WebDAV compatible sync app, there should be plenty to choose from).
